rewatch/src/helpers.rs

```diff
@@ -249,26 +250,41 @@ pub fn get_bsc() -> PathBuf {
         .to_stripped_verbatim_path()
 }
 
-pub fn get_rescript_legacy(project_context: &ProjectContext) -> PathBuf {
+pub fn get_runtime_path(project_context: &ProjectContext) -> anyhow::Result<PathBuf> {
+    let resolver = Resolver::new(ResolveOptions::default());
```

Comment:
Is the resolver expensive to make?
If this helper called multiple times, it might make sense to have this outside the function and scoped to the lifetime of the process.

Reply:
I don't know, but it is only once called in `initialize_build` if I am not mistaken.

Reply:
Okay, we should be mindful of it if we end up using it again for other parts.

There is a part of hoping it can be used in `helpers::try_package_path` as well (used by `packages::read_dependency` and `config::flatten_ppx_flags`).
In there, we are trying some logical things, yet perhaps we can offload it as well to oxc.